src/hermes/commands/deposit/invenio.py

```diff
@@ -68,6 +69,13 @@ def prepare_deposit(click_ctx: click.Context, ctx: CodeMetaContext):
     license = _get_license_identifier(ctx, licenses_api_url)
     ctx.update(invenio_path["license"], license)
 
+    communities_api_path = invenio_config.get("api_paths", {}).get(
+        "communities", _DEFAULT_COMMUNITIES_API_PATH
+    )
+    communities_api_url = f"{site_url}/{communities_api_path}"
+    communities = _get_community_identifiers(ctx, communities_api_url)
+    ctx.update(invenio_path["communities"], communities)
```

Comment:
**nitpick:** I smell some duplicate code here... Probably fine for now, might be subject to further abstraction later on.

Reply:
Yes, I noticed this, too. Even in the `_get_community/license_identifier(s)` there are some duplications